From e32253f30875b57d5895ff5a22bcee5584122e6d Mon Sep 17 00:00:00 2001
From: Joyce Ooi <joyce.ooi@intel.com>
Date: Tue, 3 Nov 2020 15:52:59 +0800
Subject: [PATCH 096/120] Revert "HSD #1508459581: ARM: dts: adjust the clock
 frequency and falling time for i2c1"

commit d9000d78adc393ca9be0b01131b918f1fec64ec4 from
https://github.com/altera-opensource/linux-socfpga.git

This reverts commit 40520ce1518d393efd70b6d80d4f19a1285129d6.
This change causes Arria10 LCD to fail although it fixes Arria10 display
port, so revert this change before finding a more appropriate solution.

Signed-off-by: Joyce Ooi <joyce.ooi@intel.com>
Integrated-by: Abhishek Paliwal <paliwal.abhishek@windriver.com>
---
 arch/arm/boot/dts/socfpga_arria10_socdk.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi b/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi
index a65fe4077049..1ff37b89b109 100644
--- a/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi
+++ b/arch/arm/boot/dts/socfpga_arria10_socdk.dtsi
@@ -135,9 +135,9 @@
 	 * adjust the falling times to decrease the i2c frequency to 50Khz
 	 * because the LCD module does not work at the standard 100Khz
 	 */
-	clock-frequency = <50000>;
-	i2c-sda-falling-time-ns = <60000>;
-	i2c-scl-falling-time-ns = <60000>;
+	clock-frequency = <100000>;
+	i2c-sda-falling-time-ns = <6000>;
+	i2c-scl-falling-time-ns = <6000>;
 
 	adc@14 {
 		compatible = "lltc,ltc2497";
-- 
2.31.1

